src/convertToObject.js

```diff
@@ -7,6 +7,14 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const mass = sourceString.split(';');
+  const result = mass.reduce((accumulator, a) => {
+    const styleElem = a.split(':').map((e) => e.trim());
+
+    return { ...accumulator, [styleElem[0]]: styleElem[1] };
+  }, {});
```

Comment:
Consider adding error handling for cases where the input string might not be formatted correctly. For example, if a segment does not contain a colon, `styleElem[1]` will be undefined, which might not be the desired behavior.